ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -767,6 +779,60 @@ func getUpdateServiceMockClient(t *testing.T,
 	return mockEcs
 }
 
+func getCreateServiceMockClient(t *testing.T,
+	ctrl *gomock.Controller,
+	taskDefinition ecs.TaskDefinition,
+	taskDefID string,
+	registerTaskDefResponse ecs.TaskDefinition,
+) *mock_ecs.MockECSClient {
+
+	mockEcs := mock_ecs.NewMockECSClient(ctrl)
+
+	createdService := &ecs.Service{
+		TaskDefinition: aws.String("arn/test-task-def"),
```

Comment:
* Since we're passing in `taskDefID`, can we set the service TaskDefinition to reflect that? ("arn/" + taskDefID)
* Then we can also assert it's the same task def passed into UpdateService() on line 822 (similar to how we're verifying CreateService() in put on lines 815-817)

Reply:
Ah, you're right - we don't pass in the task definition if it's the same as what's present in the existing service. Thanks for clarifying!